spark/dl/src/main/scala/com/intel/analytics/bigdl/example/MLPipeline/DLEstimatorLeNet.scala

```diff
@@ -66,41 +60,32 @@ object DLEstimatorLeNet {
       val model = LeNet5(classNum = 10)
 
       val trainSet = DataSet.array(load(trainData, trainLabel), sc) ->
-        BytesToGreyImg(28, 28) -> GreyImgNormalizer(trainMean, trainStd) -> GreyImgToBatch(
-        param.batchSize)
+        BytesToGreyImg(28, 28) -> GreyImgNormalizer(trainMean, trainStd)
 
       val validationSet = DataSet.array(load(validationData, validationLabel), sc) ->
-        BytesToGreyImg(28, 28) -> GreyImgNormalizer(testMean, testStd) -> GreyImgToBatch(
-        param.batchSize)
+        BytesToGreyImg(28, 28) -> GreyImgNormalizer(testMean, testStd)
 
       val dataFrameRDD : RDD[MinibatchData[Float]] = trainSet.
         asInstanceOf[DistributedDataSet[TensorMiniBatch[Float]]].data(false).map(batch => {
           val feature = batch.getInput.asInstanceOf[Tensor[Float]]
           val label = batch.getTarget.asInstanceOf[Tensor[Float]]
-          val estimatorData = MinibatchData[Float](feature.storage().array(),
-            label.storage().array())
-           estimatorData
+          MinibatchData[Float](feature.storage().array(), label.storage().array())
```

Comment:
Thanks @wzhongyuan , I added -> GreyImgToBatch(1) to the transformers and changed the name MinibatchData to just Data, so actually each record still just contains 1 image. But the code here may use some further simplification. Feel free to send a PR afterwards. We need to find a better way to convert from BigDL data objects to Spark/Scala data objects.